gallery/visualtests/nxTransferList.js

```diff
@@ -47,8 +49,14 @@ describe('NxTransferList', function() {
   });
 
   it('handles overflowing content with a tooltip on items initially scrolled out of view', async function() {
-    const [list, lastItem] = await waitAndGetElements(simpleListSelector, lastItemSelector);
+    const [list] = await waitAndGetElements(simpleListSelector);
+    await scrollIntoView(list);
+
+    const [listHalf] = await waitAndGetElements(listSelector);
+    await listHalf.evaluate(e => e.scrollBy(0, 10000));
+    await wait(500);
```

Comment:
This `wait` is probably unnecessary; doesn't `waitAndGetElements` already give the element some time to appear?

Reply:
> This `wait` is probably unnecessary; doesn't `waitAndGetElements` already give the element some time to appear?

It is needed, If I remove the test fails